src/main/kotlin/org/rust/ide/refactoring/extractEnumVariant/RsExtractEnumVariantProcessor.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring.extractEnumVariant
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.search.searches.ReferencesSearch
+import com.intellij.refactoring.BaseRefactoringProcessor
+import com.intellij.usageView.BaseUsageViewDescriptor
+import com.intellij.usageView.UsageInfo
+import com.intellij.usageView.UsageViewDescriptor
+import org.rust.ide.inspections.import.RsImportHelper
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.RsReferenceElement
+import org.rust.lang.core.psi.ext.descendantOfTypeStrict
+import org.rust.lang.core.psi.ext.parentEnum
+
+class RsExtractEnumVariantProcessor(project: Project,
+                                    private val editor: Editor,
+                                    private val ctx: RsEnumVariant) : BaseRefactoringProcessor(project) {
```

Comment:
Let's reformat it according to kotlin [style guide](https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting)